d2common/d2data/d2datadict/monstats.go

```diff
@@ -690,7 +690,7 @@ func LoadMonStats(file []byte) { // nolint:funlen // Makes no sense to split
 	d := d2common.LoadDataDictionary(file)
 	for d.Next() {
 		record := &MonStatsRecord{
-			Key:                            d.String("Id"),
+			Key:                            d.String("ID"),
```

Comment:
we should not be altering the name of the column  we are retrieving from the txt files